Incorrect ValueRenderer selection for expected_output when building eval report table

### Initial Checks

- [x] I confirm that I'm using the latest version of Pydantic AI
- [x] I confirm that I searched for my issue in https://github.com/pydantic/pydantic-ai/issues before opening this issue

### Description

In `ReportCaseRenderer.build_row` there is a typo :

Actual :
https://github.com/pydantic/pydantic-ai/blob/dd22595464ea430d9fcea388e506bf0a1697a9a4/pydantic_evals/pydantic_evals/reporting/__init__.py#L563-L564

Expected
````python
if self.include_expected_output:
            row.append(self.output_renderer.render_value(None, case.expected_output) or EMPTY_CELL_STR)
````

### Example Code

```Python

```

### Python, Pydantic AI & LLM client version

```Text
Latest
```

Sure but I am stuck : https://pydanticlogfire.slack.com/archives/C083V7PMHHA/p1746723117913459

Could not get the pre-commit validation pass.
`make` seems to pass well. 
Any idea ? It complains about missing field `version` the `uv.lock`

I just clone my fork, install precommit and make install.

```bash
lionel@Mac pydantic-ai % make
uv run ruff format
176 files left unchanged
uv run ruff check --fix --fix-only
uv run ruff format --check
176 files already formatted
uv run ruff check
All checks passed!
PYRIGHT_PYTHON_IGNORE_WARNINGS=1 uv run pyright
0 errors, 0 warnings, 0 informations 
uv run coverage run -m pytest
```


Git Log
```
> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -
don't commit to branch...................................................Passed
check yaml...........................................(no files to check)Skipped
check toml...........................................(no files to check)Skipped
fix end of files.........................................................Passed
trim trailing whitespace.................................................Passed
Fix smartquote characters................................................Passed
Normalize irregular space characters to "space"..........................Passed
Fix ligature characters with NFKD normalization..........................Passed
Format...................................................................Failed
- hook id: format
- exit code: 2

uv run ruff format
error: Failed to parse `uv.lock`
  Caused by: TOML parse error at line 2801, column 1
     |
2801 | [[package]]
     | ^^^^^^^^^^^
missing field `version`
make: *** [format] Error 2

Lint.....................................................................Failed
- hook id: lint
- exit code: 2

uv run ruff format --check
error: Failed to parse `uv.lock`
  Caused by: TOML parse error at line 2801, column 1
     |
2801 | [[package]]
     | ^^^^^^^^^^^
missing field `version`
make: *** [lint] Error 2

Lint JS..............................................(no files to check)Skipped
Typecheck................................................................Failed
- hook id: typecheck
- exit code: 2

PYRIGHT_PYTHON_IGNORE_WARNINGS=1 uv run pyright
error: Failed to parse `uv.lock`
  Caused by: TOML parse error at line 2801, column 1
     |
2801 | [[package]]
     | ^^^^^^^^^^^
missing field `version`
make: *** [typecheck-pyright] Error 2
```

codespell................................................................Passed

--- a/pydantic_evals/pydantic_evals/reporting/__init__.py
+++ b/pydantic_evals/pydantic_evals/reporting/__init__.py
@@ -558,10 +558,10 @@ class ReportCaseRenderer:
             row.append(self.input_renderer.render_value(None, case.inputs) or EMPTY_CELL_STR)
 
         if self.include_metadata:
-            row.append(self.input_renderer.render_value(None, case.metadata) or EMPTY_CELL_STR)
+            row.append(self.metadata_renderer.render_value(None, case.metadata) or EMPTY_CELL_STR)
 
         if self.include_expected_output:
-            row.append(self.input_renderer.render_value(None, case.expected_output) or EMPTY_CELL_STR)
+            row.append(self.output_renderer.render_value(None, case.expected_output) or EMPTY_CELL_STR)
 
         if self.include_output:
             row.append(self.output_renderer.render_value(None, case.output) or EMPTY_CELL_STR)

